packages/kbn-eslint-plugin-packages/src/rules/no_naked_eslint_disable.test.ts

```diff
@@ -0,0 +1,142 @@
+      sourceType: 'module',
+      ecmaVersion: 2018,
+      ecmaFeatures: {
+        jsx: true,
+      },
+    },
+  }),
+] as const;
+
+const babelTester = [
+  '@babel/eslint-parser',
+  new RuleTester({
+    parser: require.resolve('@babel/eslint-parser'),
+    parserOptions: {
+      sourceType: 'module',
+      ecmaVersion: 2018,
+      requireConfigFile: false,
+      babelOptions: {
+        presets: ['@kbn/babel-preset/node_preset'],
+      },
+    },
+  }),
+] as const;
+
+for (const [name, tester] of [tsTester, babelTester]) {
+  describe(name, () => {
+    tester.run('@kbn/packages/no_naked_eslint_disable', NoNakedESLintDisableRule, {
+      valid: [
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            // eslint-disable no-var
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            // eslint-disable-next-line no-use-before-define
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            // eslint-disable-line no-use-before-define
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            /* eslint-disable no-var */
+          `,
+        },
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            /* eslint-disable no-console, no-control-regex*/
+          `,
+        },
+      ],
+
+      invalid: [
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            /* eslint-disable */
+            const a = 1;
+          `,
+          errors: [
+            {
+              line: 1,
+              message:
+                'Using a naked eslint disable is not allowed. Please specify the specific rules to disable.',
+            },
+          ],
+          output: '\nconst a = 1;',
+        },
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            // eslint-disable-next-line
+            const a = 1;
+          `,
+          errors: [
+            {
+              line: 1,
+              message:
+                'Using a naked eslint disable is not allowed. Please specify the specific rules to disable.',
+            },
+          ],
+          output: `\nconst a = 1;`,
+        },
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            /* eslint-disable */
+          `,
+          errors: [
+            {
+              line: 1,
+              message:
+                'Using a naked eslint disable is not allowed. Please specify the specific rules to disable.',
+            },
+          ],
+          output: '',
+        },
+        {
+          filename: 'foo.ts',
+          code: fmt`
+            // eslint-disable-next-line
+          `,
+          errors: [
+            {
+              line: 1,
+              message:
+                'Using a naked eslint disable is not allowed. Please specify the specific rules to disable.',
+            },
+          ],
+          output: '',
+        },
```

Comment:
We need a test like this but for `// eslint-disable-line`, this format isn't tested and doesn't work right.

Reply:
👍